e2e/e2e_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"testing"
 
 	"github.com/hashicorp/go-multierror"
+	"istio.io/istio/pkg/spiffe"
```

Comment:
do not depend on the istio package, it makes the binary significantly larger. You can just implement the code to return the spiffe identity, just one line.